ingest_python_scripts/prechecks.py

```
@@ -0,0 +1,1119 @@
+    print(f"  [CAUTION] {msg}")
+
+def bad(msg):
+    print(f"  [BAD]     {msg}")
+
+def info(msg):
+    print(f"  {msg}")
+
+# ---------- AWS CLI wrappers ----------
+def aws(args, capture=True):
+    """Run an aws CLI command. Returns stdout text (stripped) on success."""
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)